src/os/bluestore/BlueFS.cc

```diff
@@ -2229,6 +2240,227 @@ int64_t BlueFS::_read_random(
   return ret;
 }
 
+void BlueFS::_wal_update_size(FileRef file) {
+  uint64_t current_size = file->fnode.size;
+  uint64_t flush_offset = current_size;
+
+  FileReaderBuffer buf(super.block_size);
+
+  uint64_t x_off = 0;
+  uint64_t ret = 0;
+
+  std::unique_lock u_lock(file->lock);
+
+  while(flush_offset < file->fnode.allocated) {
+    // read first part of wal flush
+    {
```

Comment:
why the extra '{'?

Reply:
I like enclosing in brackets what could be a function but it's not worth to create now